Maestro/subscriptions.json

```diff
@@ -577,10 +577,13 @@
         "https://github.com/dotnet/cli/blob/release/**/*",
         "https://github.com/dotnet/corefx/blob/master/**/*",
         "https://github.com/dotnet/corefx/blob/release/**/*",
+        "https://github.com/dotnet/corefx/blob/feature/utf8string/*",
```

Comment:
`*` doens't work, `**/*` is actually specifically handled:
https://github.com/dotnet/versions/blob/efb1bc6623cbcac8c659bd0bf188e64535a38cf4/Maestro/src/Microsoft.DotNet.Maestro.WebApi/Models/SubscriptionsModel.cs#L89
https://github.com/dotnet/versions/issues/207 related for more flexibility.

Reply:
Dangit...just remove then?

Reply:
Just replace the `*`s with `**/*` and this change LGTM.